prereise/gather/demanddata/transportation_electrification/immediate.py

```diff
@@ -0,0 +1,194 @@
+# DELETED: comm_type, LOAD, vmtsc
+
+import mileage
+import charging

```

Comment:
What is this `charging` module? It doesn't seem to be present in this branch.

Reply:
The charging module contains the helper functions that Meg and I have previously mentioned. This will be in Pull Request soon. Expect Monday next week.